.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# Summary
+
+## Issue
+
+<!-- Some issue -->
+
+## Changes
+
+<!-- Ipsum lorem. -->
```

Comment:
```suggestion
<!--Provide a detailed summary of the changes you have made. This could be in the form of bullet points, detailing the changes to each file, or a paragraph form. -->
```